src/main/java/fatec/labdev/projeto/pollingapp/poll/model/Poll.java

```diff
@@ -36,8 +36,11 @@
     @Column(name = "description")
     private String description;
 
+    @Column(name = "is_enabled")
+    private boolean isEnabled;
+
     @OneToMany(mappedBy = "poll", orphanRemoval = true, cascade = CascadeType.ALL)
-    private List<Option> options = new ArrayList<>();
+    private Set<Option> options = new HashSet<>();
```

Comment:
Tem problema em deixar criar opções repetidas?

Reply:
Não tem como criar duplicadas por causa do id diferente sempre, mas quando é Set o Jpa exclui automaticamente e usa umas querys mais eficientes.
https://dzone.com/articles/why-set-is-better-than-list-in-manytomany